server/methods/core/email.js

```diff
@@ -0,0 +1,75 @@
+import { Meteor } from "meteor/meteor";
+import { check } from "meteor/check";
+import { Jobs, Packages } from "/lib/collections";
+import { Logger, Reaction } from "/server/api";
+
+Meteor.methods({
```

Comment:
Could be declaring and exporting Meteor methods, I feel like this should be our new standard, but we've only implemented in a few new packages. however, the benefits are worth doing this as we go.. thoughts?